packaging/test_irods_client_rest_cpp.py

```diff
@@ -350,6 +350,34 @@ def test_list_with_accoutrements(self):
                 os.remove(file_name)
                 admin.assert_icommand(['irm', '-f', file_name])
 
+    def test_query_handles_case_insensitivity__issue124():
```

Comment:
This test does not work because it needs an argument as a method of this test class. `self` on 376 is not defined.

Reply:
Good catch, Alan/Codacy!